examples/hybrid/classify_case.jl

```diff
@@ -0,0 +1,10 @@
+is_baro_wave(parsed_args) = all((
+    parsed_args["config"] == "sphere",
+    parsed_args["forcing"] == nothing,
+))
+
+is_column_radiative_equilibrium(parsed_args) = all((
+    parsed_args["config"] == "column",
+    parsed_args["forcing"] == nothing,
+    parsed_args["rad"] ≠ nothing,
```

Comment:
I prefer `!=` than `≠`. A lot of us has experienced unresolved unicodes in certain browser/IDE setups.